Add indexing pressure documentation (#59456)
elastic/elasticsearch: docs/reference/cluster/nodes-stats.asciidoc

  `http`::
      HTTP connection information.

  `indexing_pressure`::

>         Indexing pressure statistics about total and current indexing load and

Reviewer: I would swap it around: current and total indexing load

(integer)
Bytes consumed by indexing requests in the coordinating or primary stage. This
value is not the sum of coordinating_bytes and primary_bytes as a node can reuse
the coordinating bytes if the primary is executed locally.

`coordinating_bytes`::
(integer)
Bytes consumed by indexing requests in the coordinating stage.

`primary_bytes`::
(integer)
Bytes consumed by indexing requests in the primary stage.

`all_bytes`::
(integer)
Bytes consumed by indexing requests in the coordinating, primary, or replica stage.

`coordinating_rejections`::
(integer)
Number of indexing requests rejected in the coordinating stage.

`primary_rejections`::
(integer)
Number of indexing requests rejected in the primary stage.

`replica_rejections`::
(integer)
Number of indexing requests rejected in the replica stage.
=======
`current`::
(object)
Contains statistics for current indexing load.
+
.Properties of `<current>`
[%collapsible%open]
=======
`combined_coordinating_and_primary_bytes`::
(integer)
Bytes consumed by indexing requests in the coordinating or primary stage. This
value is not the sum of coordinating_bytes and primary_bytes as a node can reuse

> the coordinating bytes if the primary is executed locally.

Reviewer: if the primary "stage" is executed locally

=======
======

[[cluster-nodes-stats-api-response-body-indexing-pressure]]
`indexing_pressure`::
(object)

> Contains indexing pressure statistics for the node.

Reviewer: maybe cross-link to the indexing pressure section here (index-modules-indexing-pressure)

+
.Properties of `indexing_pressure`
[%collapsible%open]
======
`total`::
(object)

> Contains statistics for cumulative indexing load since the node started.

Reviewer 2: ```suggestion
Contains statistics for the cumulative indexing load since the node started.
```

+
.Properties of `<total>`
[%collapsible%open]
=======
`coordinating_and_primary_bytes`::
(integer)
Bytes consumed by indexing requests in the coordinating or primary stage.

`replica_bytes`::
(integer)
Bytes consumed by indexing requests in the replica stage.

`all_bytes`::
(integer)

> Bytes consumed by indexing requests in the coordinating, primary, or replica stage.

Reviewer 2: Are human readable versions of these stats available?

Reviewer 3: There is not. My understanding is that human readable versions is disabled by default. So I think we can add human readable interpretations in 7.10? @ywelsch I assume it is too late to make this change after feature freeze since it is kind of a feature?

`coordinating_and_primary_memory_limit_rejections`::
(integer)

> Rejections of indexing requests in the coordinating or primary stage.

Reviewer 2: ```suggestion
Number of indexing requests rejected in the coordinating or primary stage.
```

> `replica_memory_limit_rejections`::
> (integer)
> Rejections of indexing requests in the replica stage.

Reviewer 2: ```suggestion
`replica_memory_limit_rejections`::
(integer)
Number of indexing requests rejected in the replica stage.
```

=======
`current`::
(object)
Contains statistics for current indexing load.
+
.Properties of `<current>`
[%collapsible%open]
=======
`coordinating_and_primary_bytes`::
(integer)
Bytes consumed by indexing requests in the coordinating or primary stage.

`replica_bytes`::
(integer)
Bytes consumed by indexing requests in the replica stage.

`all_bytes`::
(integer)

> Bytes consumed by indexing requests in the coordinating, primary, or replica stage.

Reviewer 2: Same question re: human readable stats
